source/index.rst

```diff
@@ -3,12 +3,47 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to open5e
-==================================
+Open5e
+======
 
-*The best open source for the SRD, and the source of the best open content*
+Open5e aims to be the best open-source resource for fifth edition d20 content.
 
-Contents:
+.. rst-class:: hide-mobile
+
+What are we about?
+------------------
+
+.. rst-class:: hide-mobile
+
+In addition to an open, readable, and easily searchable resource for the 5e SRD, we also intend to provide:
+
+.. rst-class:: hide-mobile
+
+* \• **Machine readable data** like monsters and spell lists, in formats like **csv, json, md, and xml**
+* \• **Useful tools** such as **random tables, encounter calculators, treasure generators, And more...**
+* \• Plus the best **OGL community-created** content on the web!
```

Comment:
But it doesn't render as a list. Have you tried using an rst list?

On Monday, February 1, 2016, Ean Moody <notifications@github.com> wrote:

> In source/index.rst
> <https://github.com/eepMoody/open5e/pull/31#discussion_r51521653>:
>
> >
> > -Contents:
> > +.. rst-class:: hide-mobile
> > +
> > +What are we about?
> > +------------------
> > +
> > +.. rst-class:: hide-mobile
> > +
> > +In addition to an open, readable, and easily searchable resource for the 5e SRD, we also intend to provide:
> > +
> > +.. rst-class:: hide-mobile
> > +
> > +* \• **Machine readable data** like monsters and spell lists, in formats like **csv, json, md, and xml**
> > +* \• **Useful tools** such as **random tables, encounter calculators, treasure generators, And more...**
> > +* \• Plus the best **OGL community-created** content on the web!
>
> This is how the theme is set up, that's why I added the bullets manually.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/eepMoody/open5e/pull/31/files#r51521653>.
>